index.html

```diff
@@ -1,171 +1,435 @@
-<html>
+<!DOCTYPE html>
+<html lang="en">
+
 <head>
-<meta charset="UTF-8">
-<!--[if IE]><meta http-equiv="X-UA-Compatible" content="IE=edge"><![endif]-->
-<meta name="viewport" content="width=device-width, initial-scale=1.0">
-<meta name="generator" content="Asciidoctor 1.5.2">
-<title>Kill Bill userguide</title>
-<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans:300,300italic,400,400italic,600,600italic%7CNoto+Serif:400,400italic,700,700italic%7CDroid+Sans+Mono:400">
-<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.1/css/bootstrap.min.css" integrity="sha384-WskhaSGFgHYWDcbwN70/dfYBj47jz9qbsMId/iRN3ewGhXQFZCSftd1LZCfmhktB" crossorigin="anonymous">
-<link rel="stylesheet" href="https://cdn.staticaly.com/gh/afeld/bootstrap-toc/v1.0.0/dist/bootstrap-toc.min.css">
-<link rel="stylesheet" href="/stylesheets/asciinema-player.css">
-<link rel="stylesheet" href="/stylesheets/kb.css">
```

Comment:
I like that the new design is now responsive and easier to read. However, it is a bit weird switching from the index.html to any page, as the design is now so different.

How hard do you think it would be to use the new `stylesheets/style.css` file for all other pages? BTW If we're getting rid of `kb.css`, the build script will have to be updated here: https://github.com/killbill/killbill-docs/blob/f3532dc1e7ddeb5139162aa886f7e391d7bc6c53/make.sh#L10